larray/inout/excel.py

```diff
@@ -185,7 +185,7 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=nan,
 
     Extract array from a given range (xlwings only)
 
-    >>> read_excel(fname, 'population_births_deaths', range='A9:E15')
```

Comment:
It is probably doable. See "pandas solution" in:
https://stackoverflow.com/questions/43327975/how-can-i-read-a-rangea5b10-and-place-these-values-into-a-dataframe-using-o
but not worth it at the moment. **Maybe** add an issue for this?
**Maybe** add a note in read_excel range argument docstring that it is not available using openpyxl engine?

Reply:
I create a new issue for this

Reply:
> Maybe add a note in read_excel range argument docstring that it is not available using openpyxl engine

It is already the case ;)
```
    range : str, optional
        Range to load the array from (only supported for the 'xlwings' engine). (...)
```